tests/integration/test_office365/test_configuration/test_invalid.py

```diff
@@ -0,0 +1,132 @@
+'''
+copyright: Copyright (C) 2015-2022, Wazuh Inc.
+
+           Created by Wazuh, Inc. <info@wazuh.com>.
+
+           This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+
+type: integration
+
+brief: The Wazuh 'office365' module allows you to collect all the logs from Office 365 using its API.
+       Specifically, these tests will check if that module detects invalid configurations and indicates
+       the location of the errors detected. The Office 365 Management Activity API aggregates actions
+       and events into tenant-specific content blobs, which are classified by the type and source
+       of the content they contain.
+
+components:
+    - office365
+
+suite: configuration
+
+targets:
+    - manager
+
+daemons:
+    - wazuh-analysisd
+    - wazuh-monitord
+    - wazuh-modulesd
+
+os_platform:
+    - linux
+
+os_version:
+    - Arch Linux
+    - Amazon Linux 2
+    - Amazon Linux 1
+    - CentOS 8
+    - CentOS 7
+    - Debian Buster
+    - Red Hat 8
+    - Ubuntu Focal
+    - Ubuntu Bionic
+
+tags:
+    - office365_configuration
+'''
+import pytest
+from pathlib import Path
+
+from wazuh_testing.constants.paths.logs import WAZUH_LOG_PATH
+from wazuh_testing.modules.modulesd.configuration import MODULESD_DEBUG
+from wazuh_testing.modules.modulesd import patterns
+from wazuh_testing.tools.monitors.file_monitor import FileMonitor
+from wazuh_testing.utils.configuration import get_test_cases_data
+from wazuh_testing.utils.configuration import load_configuration_template
+from wazuh_testing.utils import callbacks
+from . import CONFIGS_PATH, TEST_CASES_PATH
+
+# Marks
+pytestmark = pytest.mark.tier(level=0)
+
+# variables
+force_restart_after_restoring = True
```

Comment:
Is this necessary?

Reply:
Fixed with https://github.com/wazuh/wazuh/pull/17666/commits/af08de8e3f7e724dfd741d9ca760e8cdc11aeb42